### ISSUE TITLE
How to include a logo on the cover page using this class?

### ISSUE THREAD

**Issue by aradnix** [KorigamiK/doc-korigamik#2]
While the documentation is being prepared, I would like to know how the command to insert the logo works because I don't understand it.

From the code, I understand that there is a command called  `\logoimage` that is programmed to use three arguments that can also be commands:
```
\logoimagepth
\logoimagescale
\logolabel
```

So, the logo command is built in this way :

```
\includegraphics[scale=\@logoimagescale]{\@logoimagepath}
    \Large {\\ \textbf{\@logolabel}}
```

But how to make this work for the user?, I'm probably overcomplicating myself, but I've tried a few things, but I can't figure out how to make it work.

**Comment by KorigamiK** [comment]
Hi, I'm glad that you are trying out my class.
You can use the commands in the following way:

The `\logoimage` command takes 3 arguments:
- `logoimagepath` - The path the the image file
- `logoimagescale` - The scale factor for the image
- `logolabel` -  The title below the logo
    
```tex
\logoimage{logo.png}{0.6}{My logo label}
``` 

It is advised that you fill out all these fields yourself or you will see the placeholder text and images.
You can leave some fields blank too if you don't want anything there or each command separately.

**Comment by KorigamiK** [comment]
As a side-note, you can try changing and experimenting with the following parameters as well:

```latex
\title{Operating Systems}
\titlelabel{Lab Report}
\author{John Doe}
\rollno{asdf0123}
\semester{4th}
\semester{4th}
\course{Summer'09}
\bottomnote{Department of Computer Science \& Engineering}
\header{Header text}
```

**Comment by KorigamiK** [comment]
`

**Comment by aradnix** [comment]
> Hi, I'm glad that you are trying out my class. You can use the commands in the following way:
> 
> The `\logoimage` command takes 3 arguments:
> 
> * `logoimagepath` - The path the the image file
> * `logoimagescale` - The scale factor for the image
> * `logolabel` -  The title below the logo
> 
> ```
> \logoimage{logo.png}{0.6}{My logo label}
> ```
> 
> It is advised that you fill out all these fields yourself or you will see the placeholder text and images. You can leave some fields blank too if you don't want anything there or each command separately.

It worked. Well, one more thing to add to the pending documentation.

**Comment by aradnix** [comment]
> As a side-note, you can try changing and experimenting with the following parameters as well:
> 
> ```
> \title{Operating Systems}
> \titlelabel{Lab Report}
> \author{John Doe}
> \rollno{asdf0123}
> \semester{4th}
> \semester{4th}
> \course{Summer'09}
> \bottomnote{Department of Computer Science \& Engineering}
> \header{Header text}
> ```

Yes, I had already realized this possibility, in any case, to give it more versatility, it would be interesting if some of these fields were optional, such as `\rollno` or one of the `\semester`s.

**Comment by KorigamiK** [comment]
Yeah, I think I can add that
